README.md

```diff
@@ -8,6 +8,8 @@ https://github.com/user-attachments/assets/8097e18e-b926-4d8b-ba14-a979e4c58bda
 
 <br/>
 
+> **📚 [Comprehensive Documentation Available](docs/)** - This README provides quick start instructions. For detailed guides, API references, architecture documentation, and development workflows, see our [complete documentation](docs/).
+
```

Comment:
Should not be at the very top, probably should be listed as a link under the Resources section, also please remove usage of all emojis